MCP-1932: examOrderingStatus controller shell

dimitri-amida opened the pull request:
# Feature Name <!-- replace this with the feature/bug name -->

## Description

<!-- you don't need to write anything here -->

### What was the problem?

see (https://amida.atlassian.net/browse/MCP-1932)

### How does this fix it?

<!-- brief description of how things will work after this PR -->

### Jira Tickets

<!-- replace "000" with ticket number in both places -->

- [MCP-1932](https://amida.atlassian.net/browse/MCP-1932)

## How to test this PR

- New endpoint

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
- [ ] If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) and
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
- [ ] If changing software behavior, post a link to the PR in Slack channel #benefits-virtual-regional-office

au2 commented on api/src/main/java/gov/va/vro/api/resources/MasResource.java:
```diff
@@ -61,4 +62,36 @@ ResponseEntity<MasClaimResponse> notifyAutomatedClaimDetails(
+            description = "Data Access Server Error",
+            content = @Content(schema = @Schema(hidden = true))),
+        @ApiResponse(
+            responseCode = "404",
```
Are we going to have 404 response really?

dimitri-amida replied:
I don't know. @rajesh-Krishnasamy ?

au2 commented on api/src/main/java/gov/va/vro/api/resources/MasResource.java:
```diff
@@ -61,4 +62,36 @@ ResponseEntity<MasClaimResponse> notifyAutomatedClaimDetails(
+            content = @Content(mediaType = "application/json"))
+      })
+  @Timed(value = "exam-ordering-status")
+  @Tag(name = "MAS Exam Ordering Status")
```
Can we just use the same tag for MAS end point for now? Maybe "MAS Integration"? You need to do the same change in OpenAPIConfiguration.java

au2 commented on domain/src/main/java/gov/va/vro/model/mas/MasOrderingStatusPayload.java:
```diff
@@ -0,0 +1,30 @@
+  @Schema(
+      description = "Exam order timestamp",
+      example =
+          "225-63-70T34:63:40.566614908642876440526533774826377332727118627060130905913+84:54")
```
Why don't we have an actual ISO date here as an example?

dimitri-amida replied:
It does look weird but I copied this exactly from the open API docs. @rajesh-Krishnasamy ?

dimitri-amida replied:
I fixed it

dimitri-amida commented:
@rajesh-Krishnasamy I updated to the latest API spec. Is there anything more to do on this ticket for Exam Order Status, or is it just the controller definition. @au2 I believe I addressed all your comments.

au2 commented:
@dimitri-amida can you resolve conflicts?

dimitri-amida commented:
@au2 conflicts resolved
